go/src/koding/kites/kloud/stack/provider/bootstrap.go

```diff
@@ -85,7 +85,8 @@ func (bs *BaseStack) bootstrap(arg *stack.BootstrapRequest) (interface{}, error)
 				}
 			} else {
 				bs.Log.Info("Creating bootstrap resources belonging to identifier '%s'", cred.Identifier)
-				bs.Log.Debug("Bootstrap template: %s", tmpl.Content)
+				bs.Log.Debug("Bootstrap template:")
```

Comment:
not an issue since these are debug statements, just to note:
it is usually better to log statements in one line than multiple ones, when the logging volume is so high it will be harder to keep track of them because others will interfere between these two